src/main/webapp/i18n/de/team.json

```
@@ -20,7 +20,7 @@
                     "name": "Name",
                     "email": "E-Mail"
                 },
-                "noOwner": "Kein Tutor",
+                "noOwner": "Kein/e Tutor:in",
```

```suggestion
                "noOwner": "Kein:e Tutor:in",
```
I'd keep the gendering consistent

---

solved via 
6b18916

---

I was not sure about the use of `/` for pronouns or articles. I changed it and used plural where possible in 39df4e1